python/cudf/cudf/core/column/string.py

```diff
@@ -684,9 +735,416 @@ def split(self, pat=None, n=-1, expand=True, **kwargs):
             n = -1
 
         kwargs.setdefault("expand", expand)
+        if pat is None:
+            pat = " "
```

Comment:
The default should be to split on whitespace, not a single space. Is this something we can support?